pebblo/app/config/config.py

```diff
@@ -0,0 +1,72 @@
+import yaml
+
+from pydantic import BaseSettings, Field
+import pathlib
+
+# Default config value
+DEFAULT_SERVICE_CONFIG = './config_default.yaml'
+dir_path = pathlib.Path().absolute()
+
+
+# Port BaseModel
+class PortConfig(BaseSettings):
+    host: str = Field(default='localhost')
+    port: int = Field(default=37000)
```

Comment:
default should be 8000 same as what we have in main branch now so that with default values behaviour does not change.

Reply:
Updated the default port number
resolving